docs/src/md/kotlin.core/declarations.md

```diff
@@ -543,6 +543,8 @@ Enum class $E$ is a special kind of class with the following properties:
 
 Enum class body uses special kind of syntax (see grammar) to declare enum entries in addition to all other declarations inside the class body.
 Enum entries have their own bodies that may contain their own declarations, similar to [object declarations][Classifier declaration].
+Enum entries don't introduce their own type, they are always considered to be of type `E`.
+As a result, no members can be accessed outside the scope of the enum entry body, other than those overriden from `E` .
```

Comment:
I'd rephrase this

> ... other than those from `E`.

as overriding does not influence whether a member is accessible or not.